src/Components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -35,30 +36,37 @@ export const PlaylistModal = () => {
           </span>
         </div>
         <div className="available-playlists">
-          <label>
-            <input
-              checked={
-                watchLater.findIndex(
-                  (v) => v._id === playlistModal.video._id
-                ) !== -1
-              }
-              type="checkbox"
-              onChange={() => {
-                if (
+          {Object.keys(playlistModal.video).length !== 0 ? (
+            <label>
+              <input
+                checked={
                   watchLater.findIndex(
                     (v) => v._id === playlistModal.video._id
                   ) !== -1
-                ) {
-                  removeFromWatchLater(playlistModal.video._id);
-                } else {
-                  addToWatchLater(playlistModal.video);
                 }
-              }}
-            />
-            Watch Later
-          </label>
+                type="checkbox"
+                onChange={() => {
+                  if (
+                    watchLater.findIndex(
```

Comment:
I have seen this first time. Then googled it. Nice use case.

Reply:
Thanks arjun